core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileSourceConfig.java

```diff
@@ -101,19 +108,24 @@
     "URI will be used. Defaults to false.")
   public Boolean filenameOnly;
 
+  @Nullable
+  @Description("Schema for the source")
+  public String inputSchema;
```

Comment:
There is already a schema property. Don't add a new one, just use the existing one.